demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -306,6 +396,14 @@ export function ProductsGrid() {
             components={{
                 Toolbar: ProductsGridToolbar,
             }}
+            componentsProps={{
+                toolbar: { exportApi, selectionModel },
+            }}
+            checkboxSelection
+            selectionModel={selectionModel}
```

Comment:
should we also add `keepNonExistentRowsSelected: true,`?

Reply:
Added: 5c8c43c53073edc72b3308331b9ce24625abbf10